File: go/calc_test.go
package calc

import "testing"

func TestAdd(t *testing.T) {
	if got := Add(3, 4); got != 7 {
		t.Errorf("Add(3, 4) = %d; want 7", got)
	}
}

func TestSub4_3(t *testing.T) {
	if got := Sub(4, 3); got != 1 {
		t.Errorf("Sub(4, 3) = %d; want 1", got)
	}
}

func TestSub3_4(t *testing.T) {
	if got := Sub(3, 4); got != -1 {
		t.Errorf("Sub(3, 4) = %d; want -1", got)
	}
}

File: java/CalcTest.java
import static org.junit.jupiter.api.Assertions.assertEquals;

import org.junit.jupiter.api.Test;

public class CalcTest {

    @Test
    void testAdd() {
        assertEquals(7, Calc.add(3, 4));
    }

    @Test
    void testSub4_3() {
        assertEquals(1, Calc.sub(4, 3));
    }

    @Test
    void testSub3_4() {
        assertEquals(-1, Calc.sub(3, 4));
    }
}
